mkdocs.yml

```diff
@@ -397,6 +398,8 @@ plugins:
       integrations/haproxy.md: xdr/features/collect/integrations/application/haproxy.md
       integrations/harfanglab.md: xdr/features/collect/integrations/endpoint/harfanglab.md
       integrations/imperva_waf.md: xdr/features/collect/integrations/cloud_and_saas/imperva_waf.md
+      integrations/infoblox_ddi.md: xdr/features/collect/integrations/network/infoblox_ddi.md
+      integrations/infoblox-ddi.md: xdr/features/collect/integrations/network/infoblox_ddi.md
```

Comment:
does the duplicate line is expected ?

Reply:
First line is about `infoblox_ddi.md` (underscore), second one `infoblox-ddi.md` (dash). BTW, these redirections won’t be used AFAIK.